bower.json

```diff
@@ -41,7 +41,8 @@
     "ngInfiniteScroll": "1.0.0",
     "pnotify": "1.3.1",
     "sticky": "*",
-    "swagger-ui": "wordnik/swagger-ui#v2.0.24"
+    "swagger-ui": "wordnik/swagger-ui#v2.0.24",
+    "smart-app-banner": "*"
```

Comment:
Right now the github repo for the plugin has no tags so bower can't reference the package via a version number. We can install the package via npm but I don't think that works with your build system at the moment?

Added a comment to an open issue for the plugin: https://github.com/kudago/smart-app-banner/issues/37